config/t8_netcdf.m4

```diff
@@ -28,11 +28,12 @@ if test "x$T8_WITH_NETCDF" != xno ; then
   LIBS="$LIBS $T8_NETCDF_LIBS"
   AC_LINK_IFELSE([AC_LANG_PROGRAM(
 [[
-]],[[
   #include <netcdf.h>
-
+  #include <netcdf_par.h>
+  #include <mpi.h>
+]],[[
   int ncid;
-  nc_open (NULL, NC_NOWRITE, &ncid);
+  nc_open_par (NULL, NC_NOWRITE, MPI_COMM_SELF, MPI_INFO_NULL, &ncid);
```

Comment:
Does this mean that we always need a parallel build of netcdf? Is there a use case where only a serial build of netcdf is available?

Reply:
I think that this is the problem why the check action fails now. The netcdf version from the repository only has `netcdf.h` and `nc_open`, but not `netcdf_par.h` and `nc_open_par`